redfish-core/lib/account_service.hpp

```diff
@@ -130,6 +132,187 @@ inline std::string getPrivilegeFromRoleId(std::string_view role)
     return "";
 }
 
+inline bool getAccountTypeFromUserGroup(std::string_view userGroup,
+                                        nlohmann::json& accountTypes)
+{
+    // set false if userGroup values are not found in list, return error
+    bool isFoundUserGroup = true;
+
+    if (userGroup == "redfish")
+    {
+        accountTypes.push_back("Redfish");
+    }
+    else if (userGroup == "ipmi")
+    {
+        accountTypes.push_back("IPMI");
+    }
+    else if (userGroup == "ssh")
+    {
+        accountTypes.push_back("HostConsole");
+        accountTypes.push_back("ManagerConsole");
+    }
+    else if (userGroup == "web")
```

Comment:
Will userGroup be set to "web" for phosphor-webui and webui-vue?

Reply:
for both, I think
because I think at a time only one get used